tests/sentry/suspect_resolutions/test_metric_correlation.py

```diff
@@ -0,0 +1,53 @@
+import random
+from unittest import mock
+
+from django.utils import timezone
+
+from sentry.models import GroupStatus
+from sentry.suspect_resolutions.metric_correlation import is_issue_error_rate_correlated
+from sentry.testutils import TestCase
+
```

Comment:
I'd like to see some tests for your custom pearson r function just to verify its spitting out what we expect it to.